src/main/java/com/nirima/jenkins/plugins/docker/builder/DockerBuilderPublisher.java

```diff
@@ -288,7 +295,7 @@ boolean run() throws IOException, InterruptedException {
             log("Docker Build Response : " + imageId);
 
             // Add an action to the build
-            run.addAction(new DockerBuildImageAction(dockerApi.getDockerHost().getUri(), imageId, tagsToUse, cleanupWithJenkinsJobDelete, pushOnSuccess));
+            run.addAction(new DockerBuildImageAction(dockerApi.getDockerHost().getUri(), imageId, tagsToUse, cleanupWithJenkinsJobDelete, pushOnSuccess, noCache, pull));
```

Comment:
This line is intolerably long now - please construct the new DockerBuildImageAction and store it in a local variable `action` and then call `run.addAction(action);`
That'll also make life easier if we ever add more fields in the future.